In-house patch to change script shebangs from /bin/sh to /bin/bash.

--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-diagnostics	2018-09-05 05:25:14.646498897 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-diagnostics	2018-09-05 05:24:26.279293684 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-defaults	2018-09-05 05:26:01.045167810 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-defaults	2018-09-05 05:25:45.591236142 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-env	2018-09-05 05:26:46.826091977 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-env	2018-09-05 05:26:30.711692967 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -e
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmqctl	2018-09-05 05:27:29.282871633 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmqctl	2018-09-05 05:27:13.928757278 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-plugins	2018-09-05 05:28:11.887095882 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-plugins	2018-09-05 05:27:56.679254488 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
--- rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-server	2018-09-05 05:29:09.523260195 +0000
+++ rabbitmq-server-3.7.8/deps/rabbit/scripts/rabbitmq-server	2018-09-05 05:28:42.778217820 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 ##  The contents of this file are subject to the Mozilla Public License
 ##  Version 1.1 (the "License"); you may not use this file except in
 ##  compliance with the License. You may obtain a copy of the License
